src/ansys/grantami/recordlists/_connection.py

```diff
@@ -892,8 +889,6 @@ def search_for_lists(
         search_results = self.list_management_api.get_record_list_search_results(
             result_resource_identifier=search_info.search_result_identifier
         )
-        if not search_results:
```

Comment:
Mypy was asking for this when I used types from serverapi-openapi. This will probably be re-added in a subsequent PR.